02-peer-protocol.md

```diff
@@ -484,7 +486,7 @@ A non-funding node (fundee):
   transaction after a timeout of 2016 blocks.
 
 From the point of waiting for `funding_locked` onward, either node MAY
-fail the channel if it does not receive a required response from the
+send an `error` and fail the channel if it does not receive a required response from the
```

Comment:
seems like a contradiction here to add that it MAY fail the channel as for what a node MUST do after sending `error` is to fail the channel (BOLT 01 line 341)